cirq-core/cirq/protocols/json_test_data/spec.py

```diff
@@ -53,6 +53,7 @@
         'ParallelGateFamily',
         'PauliInteractionGate',
         'PauliStringPhasor',
+        'PauliStringPhasorGate',
```

Comment:
Since `dense_pauli_string` is serializable, it'll be trivial to make `PauliStringPhasorGate` serializable so let's do that in this PR and add the corresponding `.json` and `.repr` files.

Reply:
Also make `PauliStringPhasor` serializable